### PR TITLE
Add backported label after a successful backport

### PR DESCRIPTION
**Pull request by m-kuhn**
Add "backported" label after a successful backport operation

:crossed_fingers:

### CODE REVIEW

File: .github/workflows/backport.yml [hunk 1]
```diff
@@ -11,10 +11,18 @@ jobs:
...
         if: github.event.pull_request.merged && ( ( github.event.action == 'closed' && contains( join( github.event.pull_request.labels.*.name ), 'backport') ) || contains( github.event.label.name, 'backport' ) )
         uses: Gaurav0/backport@v1.0.26
         with:
           bot_username: qgis-bot
           bot_token: ddbdec32940df79f1adf2369b4b10f10b5a66f65
           bot_token_key: a1b2c3d47311f8e29e204f85a81b4df4a44e252c
           github_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Backported Label
+        if: steps.backport.conclution == 'success'
```
Comment (elpaso) [inline .github/workflows/backport.yml:24]:
```suggestion
        if: steps.backport.conclusion == 'success'
```
??

Comment (m-kuhn) [reply .github/workflows/backport.yml]: grazie

**Comment by roya0045** [comment]
Are the tag added to the initial PR or is this for the backport PRs?

**Comment by m-kuhn** [comment]
The initial one.
So you can search through prs that have a backport label but lack the backported label to check for things to do manually.

**Pull request merged**